superset/assets/src/SqlLab/main.less

```diff
@@ -60,7 +61,7 @@ body {
 }
 
 .Workspace .btn-sm {
-    box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.1);
+    box-shadow: 1px 1px 2px fade(@darkest, 10%);
```

Comment:
instead of all these `fade(@darkest`s do you think we should instead define all the valid versions of translucent black in the variables file, thus limiting to only certain standard transparency stops?

Reply:
Definitely agree on standardizing transparency stops... but was thinking that might be a second set of variables, so you could do `fade(@darkest, @thin)` or `fade(@someColor, @thick), etc.

Reply:
Just added 4 opacity stops that we can use with any color(s). I think this will suffice for pretty much any use case. I'm thinking I'll do a similar treatment for tint/shade stops as well, since we'd like to use them in future theming applications (e.g. tints/shades of the brand color)